src/visualizr/anitalker/dataset.py

```
@@ -1,281 +1,298 @@
+                        _db_name,
+                        f"{clip_name}.wav",
+                    ),
+                    "yaw_pitch_roll_path": path.join(
+                        pose_prefix,
+                        _db_name,
+                        "raw_videos_pose_yaw_pitch_roll",
+                        f"{clip_name}.npy",
+                    ),
+                }
+
+                if not path.exists(item_dict["yaw_pitch_roll_path"]):
```

Ruff (0.12.2)

`os.path.exists()` should be replaced by `Path.exists()`

(PTH110)